src/utils/response.js

```diff
@@ -35,3 +35,17 @@ export const errorKey404 = (i18nRoot, action, response) =>
   notFound(response)
     ? `${i18nRoot}.messages.not-found`
     : errorKey(i18nRoot, action, response);
+
+export const openBlob = blob => {
+  const pdfUrl = window.URL.createObjectURL(blob);
+  const link = document.createElement('a');
+  link.href = pdfUrl;
+  link.target = '_blank';
+  document.body.appendChild(link);
+  link.click();
+
+  setTimeout(() => {
```

Comment:
Jännittävä timeout. Onko tämä workaround johonkin?

Reply:
joo revoke ei toimi firefox:lla välittömästi tulee virhe:

Uncaught Error: Access to 'blob:https://private.testi.energiatodistusrekisteri.fi/b880e88b-0c46-4de1-a2ac-a0388c4cdd04' from script denied

Reply:
se toimii vähän ajan päästä - itselläni 0 toimii myös mutta laitoin tuon 1000 koska ei oikeastaan ole väliä milloin toi siivous tehdään

Reply:
näissä pitäisi oikeastaan miettiä että kannattaako post:ia koskaan käyttää lataukseen vai koodako tarvittavan datan polkuparametriin base64 jsonina

Reply:
koko tämä kikkailu tulee siis että lataus on post jolloin linkki a ei toimi suoraan ladattavaan sisältöön

Reply:
post:in response pitää ensin ladata selaimeen siitä muodostaa object url jota sitten voidaan klikata ohjelmallisesti